packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/SampleModuleCPP.h

```diff
@@ -112,6 +112,34 @@ struct SampleModuleCPP {
     }
   }
 
+#pragma endregion
+
+#pragma region Events
+
+  REACT_EVENT(TimedEvent);
+  std::function<void(int)> TimedEvent;
+
+  REACT_METHOD(StartTimedEvent);
+  int StartTimedEvent(int intervalMS) noexcept {
+    DebugWriteLine("StartTimedEvent", intervalMS);
+
+    int id = _timers.size();
+
+    _timers.push_back(winrt::Windows::System::Threading::ThreadPoolTimer::CreatePeriodicTimer(
+        [this, id](const winrt::Windows::System::Threading::ThreadPoolTimer &timer) {
```

Comment:
>this [](start = 9, length = 4)

It will cause crashes after the native module is deleted and the periodic timer continuing to fire events.

Reply:
We need to stop all timers in the destructor.

---
In reply to: [340890093](https://github.com/microsoft/react-native-windows/pull/3561#discussion_r340890093) [](ancestors = 340890093)

Reply:
Fixed.